pontoon/messaging/static/js/messaging.js

```diff
@@ -16,10 +16,24 @@ $(function () {
 
   const container = $('#main .container');
 
-  function isValidForm($form, isNotification, isEmail, subject, body) {
+  function isValidForm() {
```

Comment:
This should have a name like `validateForm()` so that it's clearer that it has side effects.